### PR TITLE
Made new "Community" Tab in Readme and added apkpure store

### PR DESCRIPTION
**Pull request by devilAPI**

### CODE REVIEW

**Comment by devilAPI** [comment]
https://apkpure.com/dev-doctor/com.github.linwoodcloud.dev_doctor

**Comment by devilAPI** [comment]
uploaded by me, straight from github

**Comment by devilAPI** [comment]
its fine and ready to merge

File: README.md [hunk 1]
```diff
@@ -44,3 +48,12 @@ Click [here](https://dev-doctor.cf) and try it. You have nothing to lose!
 
 * [Stable](https://dev-doctor.cf)
 * [Preview](https://preview.dev-doctor.cf)
+
+## Community
```
Comment (CodeDoctorDE) [inline README.md:52]: Okay, I think we don't need this headline any more.

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]

**Comment by devilAPI** [comment]
did it

**Comment by CodeDoctorDE** [comment]
I think you removed the wrong section? On this readme, the ios headline belongs to the website links?!

File: README.md [hunk 2]
```diff
@@ -33,14 +33,22 @@ Click [here](https://dev-doctor.cf) and try it. You have nothing to lose!
...
+Community:
 
-> Currently not available.
+* ApkPure: <https://apkpure.com/dev-doctor/com.github.linwoodcloud.dev_doctor>
 
-* Stable
-* Preview
+### iOS
 
-### Web
```
Comment (CodeDoctorDE) [inline README.md:43]: here you see what i mean

File: README.md [hunk 3]
```diff
@@ -33,14 +33,22 @@ Click [here](https://dev-doctor.cf) and try it. You have nothing to lose!
...
+> Currently not available.
 
 * [Stable](https://dev-doctor.cf)
 * [Preview](https://preview.dev-doctor.cf)
+
+## Community
+
+### Installation
```
Comment (CodeDoctorDE) [inline README.md:49]: this was my issue. we don't need an installation section

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]

**Comment by devilAPI** [comment]
I'm sorry but i have zero clue what you want.

**Comment by CodeDoctorDE** [comment]
You removed the web headline

**Comment by CodeDoctorDE** [comment]
image

**Comment by devilAPI** [comment]
fixed

File: README.md [hunk 4]
```diff
@@ -33,6 +33,10 @@ Click [here](https://dev-doctor.cf) and try it. You have nothing to lose!
 * SkyDroid: <https://to.skydroid.app/com.github.linwoodcloud.dev_doctor.izzyondroid>
 * FDroid: <https://www.f-droid.org/en/packages/com.github.linwoodcloud.dev_doctor/>
 
+Community:
```
Comment (CodeDoctorDE) [inline README.md:36]: If you add this as a headline, I can approve it! :)
`#### Community`

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]

**Comment by devilAPI** [comment]
uhh did that

**Review by CodeDoctorDE — APPROVED** [review]
👍

**Pull request merged**